docs/contributing/coding-style.md

```diff
@@ -264,6 +264,30 @@ func (s *something) SetSignedTok(st string) {
 }
 ```
 
+### Unused Variables
+
+An unused variable may increase the complexity of the source code, it may confuse the developer hence introduce a new bug.
+So please delete the unused variable.
+
+Generally, the unused variable should be reported during compilation, but in some cases, the compiler may not report an error. This is an example of the unused variable declaration that does not cause a compilation error.<br />
+
+```go
+type server struct {
+    addr string
+    port int
+}
+
+srv := &server {
+    addr: "192.168.33.10:1234",
+    // port <- unused variables
```

Comment:
I'm not sure this line 😕 
Could you please describe about this line?

Reply:
This variable represents unused variable.
If you do not use this field,  you can delete the port definition in the field struct. In other words, it means to erase the unused value.

Reply:
Do you mean the variable which is commented out will be ignored Go compilation?
The description what I want to is what cause in this situation.